recipes/credstash/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "credstash" %}
+{% set version = "1.15.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 814560f99ae2409e2c6d906d878f9dadada5d1d0a950aafb6b2c0d535291bdfb
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - credstash = credstash:main
+
+requirements:
+  host:
+    - python
+    - pip
+    - nose >=1.3.6
+  run:
+    - python
+    - cryptography >=2.1
+    - boto3 >=1.1.1
```

From the `setup.py` there is a version constraint for an optional dependency:
```suggestion
    - boto3 >=1.1.1
  run_constrained:
    - pyyaml >=3.10
```